sdcm/stress_thread.py

```diff
@@ -136,10 +136,24 @@ def create_stress_cmd(self, node, loader_idx, keyspace_idx):
 
     @staticmethod
     def _add_errors_option(stress_cmd: str, to_add: list):
-        to_add = (' '.join(to_add))
-        if ' -errors' not in stress_cmd:
-            return stress_cmd + ' -errors ' + to_add
-        return re.sub('-errors([ ]+[^-][a-zA-Z0-9-]+)+([ ]* -[a-z]+|[ ]*)', '-errors\\1 ' + to_add + '\\2', stress_cmd)
+        options = stress_cmd.split(' -')
```

Comment:
please add docstring explaining what this method does.

Reply:
Done